op-chain-ops/interopgen/recipe.go

```diff
@@ -0,0 +1,261 @@
+			L2GenesisBlockDeployConfig: genesis.L2GenesisBlockDeployConfig{
+				L2GenesisBlockGasLimit:      30_000_000,
+				L2GenesisBlockBaseFeePerGas: (*hexutil.Big)(big.NewInt(params.InitialBaseFee)),
+			},
+			OwnershipDeployConfig: genesis.OwnershipDeployConfig{
+				ProxyAdminOwner:  l2ProxyAdminOwner,
+				FinalSystemOwner: l1ProxyAdminOwner,
+			},
+			L2VaultsDeployConfig: genesis.L2VaultsDeployConfig{
+				BaseFeeVaultRecipient:                    baseFeeVaultRecipient,
+				L1FeeVaultRecipient:                      l1FeeVaultRecipient,
+				SequencerFeeVaultRecipient:               sequencerFeeVaultRecipient,
+				BaseFeeVaultMinimumWithdrawalAmount:      (*hexutil.Big)(Ether(10)),
+				L1FeeVaultMinimumWithdrawalAmount:        (*hexutil.Big)(Ether(10)),
+				SequencerFeeVaultMinimumWithdrawalAmount: (*hexutil.Big)(Ether(10)),
+				BaseFeeVaultWithdrawalNetwork:            "remote",
+				L1FeeVaultWithdrawalNetwork:              "remote",
+				SequencerFeeVaultWithdrawalNetwork:       "remote",
+			},
+			GovernanceDeployConfig: genesis.GovernanceDeployConfig{
+				EnableGovernance: false,
+			},
+			GasPriceOracleDeployConfig: genesis.GasPriceOracleDeployConfig{
+				GasPriceOracleBaseFeeScalar:     1368,
+				GasPriceOracleBlobBaseFeeScalar: 810949,
+			},
+			GasTokenDeployConfig: genesis.GasTokenDeployConfig{
+				UseCustomGasToken: false,
+			},
+			OperatorDeployConfig: genesis.OperatorDeployConfig{
+				P2PSequencerAddress: sequencerP2P,
+				BatchSenderAddress:  batcher,
+			},
+			EIP1559DeployConfig: genesis.EIP1559DeployConfig{
+				EIP1559Elasticity:        6,
+				EIP1559Denominator:       50,
+				EIP1559DenominatorCanyon: 250,
+			},
+			UpgradeScheduleDeployConfig: genesis.UpgradeScheduleDeployConfig{
+				L2GenesisRegolithTimeOffset: new(hexutil.Uint64),
+				L2GenesisCanyonTimeOffset:   new(hexutil.Uint64),
+				L2GenesisDeltaTimeOffset:    new(hexutil.Uint64),
+				L2GenesisEcotoneTimeOffset:  new(hexutil.Uint64),
+				L2GenesisFjordTimeOffset:    new(hexutil.Uint64),
+				L2GenesisGraniteTimeOffset:  new(hexutil.Uint64),
+				L2GenesisInteropTimeOffset:  new(hexutil.Uint64),
+				L1CancunTimeOffset:          new(hexutil.Uint64),
+				UseInterop:                  true,
+			},
+			L2CoreDeployConfig: genesis.L2CoreDeployConfig{
+				L1ChainID:                 l1ChainID,
+				L2ChainID:                 l2ChainID,
+				L2BlockTime:               2,
+				FinalizationPeriodSeconds: 2, // instant output finalization
+				MaxSequencerDrift:         300,
+				SequencerWindowSize:       200,
+				ChannelTimeoutBedrock:     120,
+				BatchInboxAddress:         batchInboxAddress,
+				SystemConfigStartBlock:    0,
+			},
+			AltDADeployConfig: genesis.AltDADeployConfig{
+				UseAltDA: false,
+			},
+		},
+		FaultProofDeployConfig: genesis.FaultProofDeployConfig{
+			UseFaultProofs:                  true,
+			FaultGameAbsolutePrestate:       common.HexToHash("0x03c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98"),
```

Comment:
What's this hex about? The other ones look obvious but I don't see any special value here.